lib/include/tick/array/sparsearray2d.h

```diff
@@ -135,6 +185,62 @@ Array2d<T> BaseArray2d<T>::as_array2d() {
     return c;
 }
 
+// Using save/load methods directory on SparseArray2d<T> does not compile simply
```

Comment:
What does this mean?

Reply:
The save/load functions for SparseArray2d are global and not member functions of the class SparseArray2d, which would not compile.

Reply:
Hum, I still don't get it sorry :). They won't compile because they are global and not member, or they need to be global and not member because otherwise they won't compile?
There is probably something I ignore about this compilation problem

Reply:
They need to be global, which is consistent with array.h save/load methods.

Reply:
Oh ok, then maybe can we cahnge comment to
> save/load methods need to be global to be consistent with array.h save/load methods.